src/Backend/Core/Engine/Navigation.php

```diff
@@ -27,6 +27,7 @@ public function __construct(KernelInterface $kernel)
     public function parse(TwigTemplate $template): void
     {
         $template->assign('navigation', $this->navigation);
+        $template->assign('activeParent', $this->navigation[$this->getSelectedKey($this->navigation)]['label']);
```

Comment:
@Katrienvh dit is niet altijd beschikbaar, op sommige pagina's geeft dit dus een error kan je dit oplossen?

Reply:
@carakas can you give an example of a page without a selected key?

Reply:
the login page in the backend for instance, those pages are the reason the tests are failing

Reply:
@Katrienvh 
```
$selectedKey = $this->getSelectedKey($this->navigation);
if ($selectedKey !== null && isset($this->navigation[$selectedKey])) {
  $template->assign('activeParent', $this->navigation[$selectedKey]['label']);
}
```
And of course add an `{% if %}{% else %}{% endif %}` in your template